example_simple_classification.py

```diff
@@ -0,0 +1,53 @@
+"""This file is used to create and compare two models on a particular dataset.
+It provides examples of reading from both csv and SQL Server. Note that this
+example can be run as-is after installing healthcare.ai. After you have
+found that one of the models works well on your data, move to Example2
+"""
+from healthcareai.simple_mode import SimpleDevelopSupervisedModel
+import pandas as pd
+import time
+
+
+def main():
+    t0 = time.time()
+
+    # CSV snippet for reading data into dataframe
+    dataframe = pd.read_csv('healthcareai/tests/fixtures/DiabetesClincialSampleData.csv', na_values=['None'])
+
+    # Drop columns that won't help machine learning
+    dataframe.drop(['PatientID', 'InTestWindowFLG'], axis=1, inplace=True)
+
+    # Look at the first few rows of your dataframe after the data preparation
+    print(dataframe.head())
+
+    # Step 1: Setup healthcareai for developing a model. This prepares your data for model building
+    hcai = SimpleDevelopSupervisedModel(dataframe=dataframe,
```

Comment:
Nit.  I think the alternative pep8 indentation style would be more readable here:

```
    hcai = SimpleDevelopSupervisedModel(
        dataframe=dataframe,
        predicted_column='ThirtyDayReadmitFLG',
        model_type='classification',
        impute=True,
        grain_column='PatientEncounterID',
        verbose=False
    )
```